west.yml

```diff
@@ -198,7 +198,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 88a7d079eed470ecc94010784931823a85ac3b39
+      revision: pull/391/head
```

Comment:
you can put the manifest change in the commit where the changes are first needed

Reply:
@decsny is there a problem with it being first? 

Which commit would you want this part of? The SOC one?